src/components/list/list.ios.scss

```diff
@@ -182,6 +182,10 @@ $list-ios-header-background-color:    transparent !default;
   background: $list-ios-header-background-color;
 }
 
+[dir="rtl"] .list-header-ios {
+  padding-right: $list-ios-header-padding-left;
```

Comment:
maybe renaming the css variable to `$list-ios-header-padding-start`?
thoughts @AmitMY @brandyscarney